templates/default/rundeck-upstart.conf.erb

```diff
@@ -0,0 +1,29 @@
+# RunDeck
```

Comment:
since this doesnt have any template logic in it, can you change it to `cookbook_file` resource and move this file to the `files/default` directory